Library/Formula/gdmap.rb

```diff
@@ -1,33 +1,31 @@
-require 'formula'
-
 class Gdmap < Formula
   desc "Tool to inspect the used space of folders"
-  homepage 'http://sourceforge.net/projects/gdmap/'
-  url 'https://downloads.sourceforge.net/project/gdmap/gdmap/0.8.1/gdmap-0.8.1.tar.gz'
-  sha1 'd97cc7c107dbaf9f3f3ed22ee6cef6172c115295'
-
-  depends_on 'pkg-config' => :build
-  depends_on 'intltool' => :build
-  depends_on 'gettext'
-  depends_on 'glib'
-  depends_on 'gtk+'
-
-  #The code depends on some GTK macros that are flagged as deprecated in the brew version of GTK.
-  #I assume they're not deprecated in normal GTK, because the config file disables deprecated GDK calls.
-  #The first patch turns off this disablement, making the code work fine as intended
-  #The second patch is to remove an unused system header import on one of the files.
-  #This header file doesn't exist in OSX and the program compiles and runs fine without it.
+  homepage "http://sourceforge.net/projects/gdmap/"
+  url "https://downloads.sourceforge.net/project/gdmap/gdmap/0.8.1/gdmap-0.8.1.tar.gz"
+  sha256 "a200c98004b349443f853bf611e49941403fce46f2335850913f85c710a2285b"
+
+  depends_on "pkg-config" => :build
+  depends_on "intltool" => :build
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "gtk+"
+
+  # The code depends on some GTK macros that are flagged as deprecated in the brew version of GTK.
+  # I assume they're not deprecated in normal GTK, because the config file disables deprecated GDK calls.
+  # The first patch turns off this disablement, making the code work fine as intended
+  # The second patch is to remove an unused system header import on one of the files.
+  # This header file doesn't exist in OSX and the program compiles and runs fine without it.
   patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!